### ISSUE TITLE
Club Member discount

### ISSUE THREAD

**Issue by matos-07** [tkrebs/ep3-bs#621]
Has anyone implemented a feature to identify whether the user is a member of the club or not and, if so, discount the final price of the booking?

**Comment by mjmpeace** [comment]
yes, there is a modification of Mr. Schiebold, which has this feature
